server/plugin/command.go

```diff
@@ -367,7 +367,18 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, err := p.API.CreatePost(post); err != nil {
+		p.API.LogWarn("error while creating post", "post", post, "error", err.Error())
+		return err.Error()
```

Comment:
We still want to report to the user that the subscription was created successfully here

```suggestion
		return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, err.Error())
```

Reply:
Sure , will add that too